titiler/api/deps.py

```diff
@@ -127,14 +136,21 @@ def __init__(
             title="Color Formula",
             description="rio-color formula (info: https://github.com/mapbox/rio-color)",
         ),
-        color_map: Optional[ColorMapName] = Query(
+        color_map: Optional[ColorMapNames] = Query(
             None, description="rio-tiler's colormap name"
         ),
+        resampling_method: ResamplingNames = Query(
+            ResamplingNames.nearest, description="Resampling method."  # type: ignore
+        ),
```

Comment:
This is just for documentation and validation. 

`resampling_method` will be passed via `kwargs`